lib/ht/search_client/test.rb

```diff
@@ -38,11 +38,11 @@ def stay_class?
       self.class.to_s =~ /Stay/
     end
 
-    def mocked_results(properties)
+    def mocked_results(properties, total)
       {
         page:     params.fetch(:page),
         per_page: params.fetch(:per_page),
-        total:    properties.length,
+        total:    total,
```

Comment:
> by default total should be [per_page, properties.length].min

wouldn't that lead to this incorrect scenario?

```
properties: [ put 20 properties here ]
page: 2
per_page: 32
total: 20
```

here is the `page` that is breaking the equation